commons/connector/altoida/AltoidaVersion.avsc

```diff
@@ -0,0 +1,9 @@
+{
+  "namespace": "radarcns.connector.altoida",
+  "type": "record",
+  "name": "AltoidaVersion",
```

Comment:
perhaps just make this a metadata topic, e.g. AltoidaMetadata with a `time`, `timeReceived` and `version` attribute, where the version is a string. The time could be derived as the first measurement made for a given zip file, or the timestamp that's in the ZIP file entry.